deploy/kubernetes/clusterrole.yaml

```diff
@@ -7,42 +7,17 @@ metadata:
     version: ${VERSION_LABEL}
 rules:
 - apiGroups: [""]
-  resources:
-  - configmaps
-  - endpoints
-  - namespaces
-  - nodes
-  - pods
-  - services
-  - replicationcontrollers
-  verbs:
-  - get
-  - list
-  - watch
+  resources: ["configmaps", "endpoints", "namespaces", "nodes", "pods", "services", "replicationcontrollers"]
```

Comment:
Is there a reason why we are changing the syntax for these lists? Does not seem to be related to the issue this PR is trying to address. Also, IIRC, @gbaufake does a diff on these files so he can catch any changes here to upstream istio and maistra. Changing the syntax may break those changes.

Also, now we use different syntax here as compared to the new stuff below which still uses "one item per line". We should make this file consistent and use the same syntax throughout.